### PR TITLE
[Draft] Testnet v0.5 docs update

### PR DESCRIPTION
**Pull request by lezzokafka**
To do list:
- [x] Provide the new genesis.json for v0.5.1 ([link](https://github.com/crypto-com/chain-docs/compare/master...lezzokafka:v0.4_docs_update?expand=1#diff-8858730c1ec093ea7a43196e843f91ccR44)); 
- [x] Update the seeds ([link](https://github.com/crypto-com/chain-docs/pull/133/files#diff-8858730c1ec093ea7a43196e843f91ccL195));
- [x] Replace the `--genesis_app_hash ` flag in the command for starting _chain-abci_  with the new genesis_app_hash ( [link_1](https://github.com/crypto-com/chain-docs/compare/master...lezzokafka:v0.4_docs_update?expand=1#diff-8858730c1ec093ea7a43196e843f91ccR100) and [link_2](https://github.com/crypto-com/chain-docs/compare/master...lezzokafka:v0.4_docs_update?expand=1#diff-8858730c1ec093ea7a43196e843f91ccR175))

### CODE REVIEW

File: docs/getting-started/thaler-testnet.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ This is an early tutorial for the developers and brave and patient super-early a
 
 ### Step 0. Install Intel SGX SDK 2.8 and other pre-requisites
 
-- Make sure your CPU supports SGX and it is enabled in BIOS. [This GitHub repository](https://github.com/ayeks/SGX-hardware) has more information about supported hardware and cloud vendors.
+- Make sure your CPU supports SGX, and it is enabled in BIOS. [This GitHub repository](https://github.com/ayeks/SGX-hardware) has more information about supported hardware and cloud vendors.
 
 - You can download the Linux SGX SDK installers from the Intel Open Source [website](https://01.org/intel-softwareguard-extensions/downloads/intel-sgx-linux-2.8-release). More details can be found in this [installation guide](https://download.01.org/intel-sgx/sgx-linux/2.8/docs/Intel_SGX_Installation_Guide_Linux_2.8_Open_Source.pdf).
```
Comment (tomtau) [inline docs/getting-started/thaler-testnet.md]: 2.9

File: docs/getting-started/thaler-testnet.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ This is an early tutorial for the developers and brave and patient super-early a
 
 ### Step 0. Install Intel SGX SDK 2.8 and other pre-requisites
```
Comment (tomtau) [inline docs/getting-started/thaler-testnet.md]: 2.9

File: docs/getting-started/thaler-testnet.md [hunk 3]
```diff
@@ -206,11 +62,14 @@ Depending on what you wish to test on the testnet, e.g. monitoring, you can refe
 
 ### Step 3. Run everything
 
+Before we move forward:
+
 - Make sure `aesmd` service is running (you may potentially start it up manually with `LD_LIBRARY_PATH=/opt/intel/libsgx-enclave-common/aesm /opt/intel/libsgx-enclave-common/aesm/aesm_service`).
```
Comment (tomtau) [inline docs/getting-started/thaler-testnet.md]: not sure if this changed in 2.9?

**Review by tomtau — CHANGES_REQUESTED** [review]

File: docs/getting-started/thaler-testnet.md (hunk 1 above)
Comment (lezzokafka) [reply docs/getting-started/thaler-testnet.md]: Updated, see [4c3840f](https://github.com/crypto-com/chain-docs/pull/133/commits/4c3840f2488de553898914e7692ce7ca8af888a2).

File: docs/getting-started/thaler-testnet.md (hunk 2 above)
Comment (lezzokafka) [reply docs/getting-started/thaler-testnet.md]: Updated, see [4c3840f](https://github.com/crypto-com/chain-docs/pull/133/commits/4c3840f2488de553898914e7692ce7ca8af888a2).

File: docs/getting-started/thaler-testnet.md [hunk 4]
```diff
@@ -23,16 +23,16 @@ has Intel PSW and other dependencies pre-installed
...
-Chain v0.4 can be [downloaded from GitHub](https://github.com/crypto-com/chain/releases/download/v0.4.0/crypto-com-chain-release-0.4.0.tar.gz).
+Chain v0.5 can be [downloaded from GitHub](https://github.com/crypto-com/chain/releases/download/v0.5.0/crypto-com-chain-release-0.5.0.tar.gz).
 
 ::: warning CAUTION
-Crypto.com Chain v0.4 is not backwards compatible with v0.3 released earlier. So, if you were running a node with the old
+Crypto.com Chain v0.5 is not backwards compatible with v0.3 nor v0.4 released earlier. So, if you were running a node with the old
 version of Crypto.com Chain, you will have to delete all the associated data.
 
-Also, please note the [released binary changes](https://github.com/crypto-com/chain/releases/tag/v0.4.0).
+Also, please note the [released binary changes](https://github.com/crypto-com/chain/releases/tag/v0.5.0).
```
Comment (tomtau) [inline docs/getting-started/thaler-testnet.md]: link 0.5.1?

File: docs/getting-started/thaler-testnet.md (hunk 3 above)
Comment (lezzokafka) [reply docs/getting-started/thaler-testnet.md]: Any comments on this? @leejw51crypto @allthatjazzleo

Comment (leejw51crypto) [reply docs/getting-started/thaler-testnet.md]: it's the same as before
```
ps ax | grep aesm -> check whether aesm service is on
ls /dev/isgx -> check whether driver is on
```
can check with above commands

**Review by leejw51crypto — APPROVED** [review]
lgtm

File: docs/getting-started/thaler-testnet.md (hunk 4 above)
Comment (lezzokafka) [reply docs/getting-started/thaler-testnet.md]: Fixed, thanks @tomtau.

File: docs/getting-started/thaler-testnet.md (hunk 3 above)
Comment (tomtau) [reply docs/getting-started/thaler-testnet.md]: @leejw51crypto I meant the path `/opt/intel/libsgx-enclave-common/aesm/`

Comment (tomtau) [reply docs/getting-started/thaler-testnet.md]: not sure if the default installation path changed

Comment (lezzokafka) [reply docs/getting-started/thaler-testnet.md]: See page 11 of the [installation guide](https://download.01.org/intel-sgx/sgx-linux/2.9.1/docs/Intel_SGX_Installation_Guide_Linux_2.9.1_Open_Source.pdf)?

**Pull request merged**